Sources/PrinceOfVersions/Common/ConfigurationData.swift

```diff
@@ -29,9 +29,9 @@ struct Requirements: Decodable {
 
     var allRequirements: [String: Any]? {
         guard let requiredOSVersion = requiredOSVersion else { return nil }
```

Comment:
We should change this logic a bit...

JSON without requred_os_version is still valid and we should take it in the count (without early return). The only thing is that the OS version will be handled internally from POV so the user is not required to add OS version check on the implementation side. (this should be mentioned in the readme as well)

Additionally, if JSON does not have requirements at all, this is handled as a "valid" configuration. I.e. in the array if the first one does not have configuration value, the second one does...the first one is valid and we should use it. 

Please apply that logic in our lib and add some test cases for that as well.